Source/WebCore/rendering/TextPainter.cpp

```diff
@@ -222,7 +222,11 @@ static bool forceUseGlyphDisplayListForTesting = false;
 
 bool TextPainter::shouldUseGlyphDisplayList(const PaintInfo& paintInfo)
 {
+#if PLATFORM(WATCHOS)
```

Comment:
Can we turn this into a `USE(...)` macro?

Reply:
Sure.